packages/backend/src/managers/application/applicationManager.ts

```diff
@@ -53,11 +53,13 @@ import {
 import { VMType } from '@shared/models/IPodman';
 import { RECIPE_START_ROUTE } from '../../registries/NavigationRegistry';
 import type { RpcExtension } from '@shared/messages/MessageProxy';
+import { TaskRunner } from '../TaskRunner';
 
 export class ApplicationManager extends Publisher<ApplicationState[]> implements Disposable {
   #applications: ApplicationRegistry<ApplicationState>;
   protectTasks: Set<string> = new Set();
   #disposables: Disposable[];
+  #taskRunner: TaskRunner;
```

Comment:
if task runner is stored like that it should be disposable, and be disposed if the class is disposed, cancelling any pending tasks it is managing ?

Reply:
That would be a good enhancement, but as this is only a refactoring, I don't want to change any behaviour